models/Program.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Enum for the program format
+ */
+export enum programFormat {
+    online = "online",
+    inPerson = "in_person",
+    blended = "blended",
+}
+
+/**
+ * Request Body Input for POST /program
+ */
+export type createProgramInput = {
```

Comment:
It looks like the TypeScript convention for defining custom types is to capitalize them https://www.typescriptlang.org/docs/handbook/2/objects.html

Reply:
Fixed!